Automattic/wp-calypso
# Use signupUrl from API to link OAuth signup

Adds a proper signup url for OAuth client signup flow(s).
~Test with D6939 server side patch.~ (deployed)
  
<img width="907" alt="screen shot 2017-08-23 at 15 40 29" src="https://user-images.githubusercontent.com/326402/29616243-82a94a54-8819-11e7-95bb-0e3d19262d70.png">

  
#### Testing instructions
  
1. Run `git checkout add/oauth-signup-url` and start your server, or open a [live branch](https://calypso.live/?branch=add/oauth-signup-url)
2. Open a OAuth client [`Login` page](http://calypso.localhost:3000/log-in?client_id=50916&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D929106ee4827adb383bf5b053741a9c1%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253Dmyaccount%26blog_id%3D0%26wpcom_connect%3D1) with a proper `redirect_to`
3. Check that you redirected to a proper page when clicking `Create an Account`.
  
#### Additional notes
  
- There  is a "lag" between page load and calypso getting the proper signup url to use on the `Create an Account` link, so I gray the link while it's loading.
  
#### Reviews
